app/locales/ml/overlap.json

```diff
@@ -0,0 +1,10 @@
+{
+  "overlap_title": "ഓവർലാപ്പ് പരിശോധിക്കുക",
+  "overlap_para_1": "പച്ച നടപ്പാത നിങ്ങളുടെ ലൊക്കേഷൻ ചരിത്രത്തെ പ്രതിനിധീകരിക്കുന്നു. ഇളം പർപ്പിൾ സർക്കിളുകൾ പൊതു ഡാറ്റാഗണത്തെ പ്രതിനിധീകരിക്കുന്നു",
```

Comment:
and two more newline characters here...
 "overlap_para_1": "The green trail represents your location history\n\nThe light purple circles represent the public dataset",
can you please add them?

Reply:
done..